package.json

```diff
@@ -143,15 +148,60 @@
     "test-test:watch": "nodemon -e js,jsx --exec npm run test-test",
-    "lint": "bin/run-lint .",
-    "css-lint": "stylelint 'client/**/*.scss' --syntax scss"
+    "lint": "run-s lint:*",
+    "lint:js": "node bin/run-lint .",
+    "lint:css": "stylelint client/**/*.scss --syntax scss",
+    "lint:config-defaults": "node server/config/validate-config-keys.js",
+    "lint:mixedindent": "mixedindentlint --ignore-comments client/**/*.scss assets/**/*.scss **/*.js **/*.jsx !build/** !.git/** !node_modules/** !public/** !client/config/index.js",
+    "eslint-branch": "node bin/eslint-branch.js",
+    "prestart": "npm run install-if-needed && check-node-version --package",
+    "start": "npm run build",
+    "poststart": "npm run env -- node build/bundle.js",
+    "prebuild": "npm run install-if-needed",
+    "build": "run-p build-devdocs:* build-server build-css",
+    "postbuild": "npm run bundle",
+    "prebuild-desktop": "npm run install-if-needed",
+    "build-desktop": "run-p build-server build-css",
+    "postbuild-desktop": "npm run bundle",
+    "prebuild-server": "mkdirp build",
+    "build-server": "npm run env -- webpack --display-error-details --config webpack.config.node.js",
+    "build-css": "run-p build-css:*",
+    "build-css:style": "node-sass --include-path client assets/stylesheets/style.scss public/style.css && npm run autoprefixer -- public/style.css && rtlcss public/style.css public/style-rtl.css",
+    "build-css:debug": "node-sass --include-path client --source-map \"public/style-debug.css.map\" assets/stylesheets/style.scss public/style-debug.css && npm run autoprefixer -- public/style-debug.css",
+    "build-css:editor": "node-sass --include-path client assets/stylesheets/editor.scss public/editor.css && npm run autoprefixer -- public/editor.css",
+    "build-css:directly": "node-sass --include-path client assets/stylesheets/directly.scss public/directly.css && npm run autoprefixer -- public/directly.css",
+    "autoprefixer": "postcss -r --use autoprefixer --autoprefixer.browsers \"last 2 versions, > 1%, Safari >= 8, iOS >= 8, Firefox ESR, Opera 12.1\"",
+    "build-devdocs:proptypes-index": "npm run env -- node server/devdocs/bin/generate-proptypes-index.js client/**/index.js client/**/index.jsx client/**/example.jsx !**/test/**",
+    "build-devdocs:components-usage-stats": "npm run env -- node server/devdocs/bin/generate-components-usage-stats.js client/**/*.js client/**/*.jsx !**/docs/** !**/test/** !**/docs-example/**",
+    "build-devdocs:index": "npm run env -- node server/devdocs/bin/generate-devdocs-index **/*.md !build/** !.git/** !node_modules/** !public/**",
+    "bundle": "npm run env -- node server/bundler/bin/bundler.js",
+    "update-deps": "npm run rm -- node_modules && npm run rm -- npm-shrinkwrap.json && npm install --no-optional && npm install --no-optional && npm shrinkwrap --dev",
+    "preshrinkwrap": "npm prune",
+    "postshrinkwrap": "node bin/trim-shrinkwrap.js",
```

Comment:
This will be automatically executed every time `npm shrinkwrap --dev` is called, so no need to worry about custom commands or `shonkwrap` anymore :)